pages/common/xz.md

```diff
@@ -2,14 +2,22 @@
 
 > Compress or decompress .xz and .lzma files.
 
-- Compress a file:
+- Compress a file to the xz file format:
 
 `xz {{file}}`
 
-- Decompress a file:
+- Decompress a xz file:
 
 `xz -d {{file.xz}}`
 
+- Compress a file to the lzma file format:
+
+`xz --format=lzma {{file}}`
+
+- Decompress a lzma file:
```

Comment:
Small tweak: I'd recommend `an` instead of `a` here. `lzma` is an abbreviation, so it sounds better :-)

```suggestion
- Decompress an lzma file:
```

Reply:
:+1: